pages/en/lb3/readmes/loopback-example-connector-soap.md

```diff
@@ -12,22 +12,25 @@ node stock-ws.js
 
 ### Example 2
 
-Get weather and forecast information for a given zip code:
+Get Periodic Table element information:
 
 ```
-node weather-rest.js
+node periodictable-rest.js
 ```
 
 ### Example 3
 
 To expose REST APIs to proxy the SOAP web services, run:
 
 ```
-node weather-ws.js
+node periodictable-ws.js
 ```
 
 then browse to [http://localhost:3000/explorer](http://localhost:3000/explorer)
 
+In API explorer's operations list, select `GET /PeriodictableServices/GetAtomicNumber` or `GET /PeriodictableServices/GetAtomicWeight`, then enter `elementName` and click on `Try it out`. 
```

Comment:
Nitpick: Change `PeriodictableServices` to `PeriodicTableServices`; it shouldn't affect the routing, but it helps a bit with readability, IMO.